src/components/material.js

```diff
@@ -81,13 +81,16 @@ module.exports.Component = registerComponent('material', {
 
   updateShader: function (shaderName) {
     var data = this.data;
-    var Shader = shaders[shaderName] && shaders[shaderName].Shader;
-    var material;
-    if (!Shader) { throw new Error('Unknown shader ' + shaderName); }
-    this.shader = new Shader();
-    this.shader.el = this.el;
-    material = this.shader.init(data);
-    this.setMaterial(material);
+    var ShaderClass = shaders[shaderName] && shaders[shaderName].Shader;
```

Comment:
I think the world class here is not a good fit. It's confusing. We don't use classes. The capitalization of the variable already indicates a constructor. It's probably the most widely spread convention in JS.